roles/benchmarking_deploy_coco_dataset/files/entrypoint.sh

```diff
@@ -5,107 +5,19 @@ set -u
 set -o pipefail
 set -x
 
-cd /storage
```

Comment:
@Maxusmusti, please keep this part from the original script, 
we want the download to work when we don't provide our private S3 secret